src/parser.rs

```diff
@@ -362,9 +438,10 @@ impl Library {
             }
         }
 
-        if attrs.by_name("is-gtype-struct").is_some() {
-            return Ok(());
-        }
+        // TODO: unsure of how to handle this
+        //if attrs.by_name("is-gtype-struct").is_some() {
+        //    return Ok(());
+        //}
```

Comment:
`Result<Option<Type>>`.... I don't know why my mind completely ignored that....
Done.